app/src/main/java/currencyConverter/model/CurrencyModel.java

```diff
@@ -0,0 +1,23 @@
+package currencyConverter.model;
+
+public class CurrencyModel {
```

Comment:
Don't like word Model in the name. Does it mean that this class describes a single currency rate from data model, which can be used in conversion calculations? If yes, maybe package name `model` should be enough?